packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.jsx

```diff
@@ -53,7 +53,8 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
         if (selected_item && selected_item.value !== value) {
             onChange({ target: { name, value: selected_item.value } });
         }
-    }, [selected_item, onChange, name, value]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we need this?

Reply:
Yes, because if we left `value` in array of dependencies switcher (between `Long` and `Short`) wouldn't work correctly. And `// eslint-disable-next-line react-hooks/exhaustive-deps` was written here in order not show tips from eslint to add `value` in that array. Thank you for noticing!